tomcat/src/main/java/org/apache/common/ContentType.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.common;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public enum ContentType {
+
+    TEXT_CSS("text/css"),
+    TEXT_HTML("text/html"),
+    APPLICATION_JS("application/js"),
+    PERMIT_ALL("*/*");
+
+    private final String value;
+
+    ContentType(String value) {
+        this.value = value;
+    }
+
+    public static ContentType of(String value) {
+        return Arrays.stream(values())
+                .filter(contentType -> Objects.equals(contentType.getValue(), value))
```

Comment:
getValue()대신 그냥 value를 쓸 수 있는걸요~
```suggestion
                .filter(contentType -> Objects.equals(contentType.value, value))
```